templates/registration/profile.html

```diff
@@ -44,7 +44,37 @@ <h2 class="profile-header">Welcome, {% firstof user.get_full_name user.get_usern
     {% else %}
         <p>You haven't invested in anything yet!</p>
         <div class="profile-boxes">
-            <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}?campaign-status=Active">START INVESTING</a>
+            <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}">START INVESTING</a>
         </div>
     {% endif %}
 {% endblock %}
+
+{% block extrajs %}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.4/Chart.min.js"></script>
+
+    <script>
+    var ctx = document.getElementById("myChart").getContext('2d');
+    var myChart = new Chart(ctx, {
+      type: 'bar',
+      data: {
+        labels: [],
+        datasets: [{
+          label: 'INVESTED',
+          data: [{{ total_investments|floatformat:2|intcomma }},],
+          backgroundColor: "#85929E"
+        }, {
+          label: 'EARNED',
+          data: [{{ total_earned|floatformat:2|intcomma }},0],
+          backgroundColor: "#34495E"
+        }]
+      }
+    });
+    </script>
+
+    <script>
+    var x = {{ total_earned|floatformat:2|intcomma }};
+    var y = {{ total_investments|floatformat:2|intcomma }};
+    var z = x / y * 100;
+    document.getElementById("percentage").innerHTML = z.toFixed(2) + ' %';
```

Comment:
Is there a load tag that I need in order for this to work? It seems to not load this on it's own